demo/src/index.js

```diff
@@ -21,15 +21,32 @@ class Demo extends Component {
                 className="ReactWebChat"
             >
                 <ReactWebChatComponent
-                    url="http://localhost:8080/echo/"
+                    url="https://dev.feersum.io/chat_sockjs"
```

Comment:
@chrislombaard Should we commit this? Ideally I would like to move this out into .env files for dev

Reply:
I keep forgetting to remove this when I push. Sorry. Will remove